### DISCUSSION TITLE
About the cluttering in just one readme.

### DISCUSSION THREAD

**Discussion by abhikumar63** [ishtms/learn-nodejs-hard-way#1] [General]
Hey, I just saw your post on reddit. I am learning Node.js for some time and this content is very good.
I just have a suggestion about all the chapters being in one readme. It will be very difficult as the content grows. You can divide the content in several readme files and hen create a link for that file.

I would be happy to do that for you.

**Comment by ishtms** [comment]
I've created the whole structure of chapters, you may check now. Looks like, the Readme is much easier to read now.

**Comment by abhikumar63** [comment]
Yeah it looks much better now. ;-)
